licenses.d2iq.yaml

```diff
@@ -470,20 +470,20 @@ resources:
       - license_path: COPYING
         ref: ${image_tag}
         url: https://github.com/ceph/ceph
-  - container_image: quay.io/fairwinds/nova:3.4.0
+  - container_image: ghcr.io/mesosphere/dkp-container-images/quay.io/fairwinds/nova:3.4.0-d2iq.0
     sources:
       - license_path: LICENSE
-        ref: ${image_tag}
```

Comment:
For the tag matching we are using the https://github.com/drone/envsubst which supports functions - https://github.com/drone/envsubst?tab=readme-ov-file#supported-functions

We can use `${var%pattern}` to remove suffix of the tag and keep it dynamic. In this particular case it would be:

```
ref: ${image_tag%-d2iq.0}
```

Reply:
updated!